### DISCUSSION TITLE
Why English Quote (') looks bad in my blog? - XX's Blog

### DISCUSSION THREAD

**Discussion by giscus** [xxchan/xxchan.github.io#55] [Announcements]
# Why English Quote (') looks bad in my blog? - XX's Blog

As you might have noticed, I write blogs in both Chinese and English. My approach to multilingual blogs is straightforward and somewhat brute-force: I simply put them together without a language switcher or any filtering. Admittedly, I haven't figured out how to implement this in Jekyll due to a lack of motivation.

https://xxchan.me/misc/2024/06/12/quotation-mark.html

**Comment by xxchan** [comment]
Wait, I should use `font-family: <English font>, <Chinese font>`, then the problem is solved 😅
